# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/libwebrtcglue/MediaConduitInterface.h
# Commit: f16cf2573a4d
# Full Hash: f16cf2573a4dafa7d909ec43a81c5379480ca994
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 5
# Description:
#   Bug 1654112 - Move AudioConduit to use upstream for audioLevel and remove unused code. r=ng
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D102276
# ==============================================================================

diff -r 90a15d97399e -r f16cf2573a4d dom/media/webrtc/libwebrtcglue/MediaConduitInterface.h
--- a/dom/media/webrtc/libwebrtcglue/MediaConduitInterface.h	Mon Jan 18 11:07:55 2021 +0100
+++ b/dom/media/webrtc/libwebrtcglue/MediaConduitInterface.h	Mon Jan 18 11:16:22 2021 +0100
@@ -11,7 +11,6 @@
 #include "mozilla/RefPtr.h"
 #include "mozilla/RefCounted.h"
 #include "mozilla/UniquePtr.h"
-#include "RtpSourceObserver.h"
 #include "RtcpEventObserver.h"
 #include "CodecConfig.h"
 #include "VideoTypes.h"
@@ -35,6 +34,9 @@
 }
 
 namespace mozilla {
+namespace dom {
+struct RTCRtpSourceEntry;
+}
 
 enum class MediaSessionConduitLocalDirection : int { kSend, kRecv };
 